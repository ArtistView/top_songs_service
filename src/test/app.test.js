/**
 * @jest-environment jsdom
 */

import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Enzyme, { shallow, render, mount } from 'enzyme';
import App from '../client/app.jsx';
import {fetch} from 'whatwg-fetch';

// it('Testing to see if Jest works', () => {
//   expect(1).toBe(1)
// })
describe('Main App', () => {
  const wrapper = shallow(<App />);

  it('App renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
})

