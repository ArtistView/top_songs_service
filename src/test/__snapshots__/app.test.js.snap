// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main App App renders correctly 1`] = `ShallowWrapper {}`;
